package/helper_scripts/check_for_default_sa.sh

```diff
@@ -2,29 +2,20 @@
 
 set -eE
 
-handle_error() {
```

Comment:
IIRC, had to add this trap handler to make sure we print false when `kubectl` command fails for some reason.